configs/reduced/rocky_85/vm_image_scanning.textproto

```diff
@@ -0,0 +1,165 @@
+benchmark_id: "journald-is-configured-to-send-logs-to-rsyslog"
+benchmark_id: "systemd-journal-remote-installed"
+benchmark_id: "systemd-journal-remote-enabled"
+benchmark_id: "journald-no-logs-from-remote-client"
+benchmark_id: "journald-service-enabled"
+benchmark_id: "journald-is-configured-to-compress-large-log-files"
+benchmark_id: "journald-write-to-persistent-disk"
+benchmark_id: "journald-default-file-permissions-configured"
+benchmark_id: "logfile-permissions"
+benchmark_id: "logrotate-configured"
+benchmark_id: "cron-daemon-enabled"
+benchmark_id: "crontab-permissions"
+benchmark_id: "cron-hourly-permissions"
+benchmark_id: "cron-daily-permissions"
+benchmark_id: "cron-weekly-permissions"
+benchmark_id: "cron-monthly-permissions"
+benchmark_id: "cron-d-permissions"
+benchmark_id: "at-cron-restricted-to-authorized-users"
+benchmark_id: "sshd-config-permissions"
+benchmark_id: "sshd-private-host-key-permissions"
+benchmark_id: "sshd-public-host-key-permissions"
+benchmark_id: "ssh-access-limited"
+benchmark_id: "ssh-loglevel-appropriate"
+benchmark_id: "ssh-pam-enabled"
+benchmark_id: "ssh-root-login-disabled"
+benchmark_id: "ssh-hostbasedauthentication-disabled"
+benchmark_id: "ssh-permitemptypasswords-disabled"
+benchmark_id: "ssh-permituserenvironments-disabled"
+benchmark_id: "ssh-ignorerhosts-enabled"
+benchmark_id: "systemwide-crypto-policy-is-not-overridden"
+benchmark_id: "ssh-warning-banner-configured"
+benchmark_id: "ssh-maxauthtries-4-or-less"
+benchmark_id: "ssh-maxstartups-configured"
+benchmark_id: "ssh-maxsessions-4-or-less"
+benchmark_id: "ssh-logingrace-one-minute-or-less"
+benchmark_id: "ssh-idle-timeout-interval-configured"
+benchmark_id: "sudo-is-installed"
+benchmark_id: "sudo-commands-use-pty"
+benchmark_id: "sudo-log-file-exists"
+benchmark_id: "reauthentication-for-privilege-escalation-not-disabled"
+benchmark_id: "sudo-authentication-timeout-is-configured-correctly"
+benchmark_id: "access-to-su-restricted"
+benchmark_id: "custom-authselect-profile-is-used"
+benchmark_id: "authselect-includes-with-faillock"
+benchmark_id: "password-creation-reqs-configured"
+benchmark_id: "failed-password-lockout-configured"
+benchmark_id: "password-reuse-limited"
+benchmark_id: "password-hashing-algorithm-sha-512"
+benchmark_id: "password-expiration-365-days-or-less"
+benchmark_id: "minimum-days-between-password-changes-7-or-more"
+benchmark_id: "password-expiration-warning-days-7-or-more"
+benchmark_id: "inactive-password-lock-30-days-or-less"
+benchmark_id: "last-password-change-date-in-past"
+benchmark_id: "system-accounts-secured"
+benchmark_id: "default-user-shell-timeout-900-or-less"
+benchmark_id: "default-group-for-root-account-is-gid-0"
+benchmark_id: "default-user-umask-027-or-more-restrictive"
+benchmark_id: "world-writable-dirs-sticky"
+benchmark_id: "etc-shadow-permissions"
+benchmark_id: "etc-group-permissions"
+benchmark_id: "etc-gshadow-permissions"
+benchmark_id: "etc-passwd-permissions"
```

Comment:
Nit: Keep same order as in instance_scanning.textproto, this should go above etc-shadow-permissions